baselines/dom.generated.d.ts

```diff
@@ -2569,6 +2569,8 @@ interface Document extends Node, GlobalEventHandlers, NodeSelector, DocumentEven
       * @param content The text and HTML tags to write.
       */
     writeln(...content: string[]): void;
+    createElement(tagName: "picture"): HTMLPictureElement;
```

Comment:
Actually the overloads wouldn't be ignored, even if they are defined after the more generic signature. The reason why the order is messed up is that the information in the json files are picked up after the emitting from the spec was done, so they are always placed at the end. I can fix it later, though it shouldn't affect the functionality now.